lib/sidekiq/job/iterable.rb

```diff
@@ -308,6 +313,10 @@ def handle_completed(completed)
           raise "Unexpected thrown value: #{completed.inspect}"
         end
       end
+
+      def mono_now
+        ::Process.clock_gettime(::Process::CLOCK_MONOTONIC)
+      end
```

Comment:
Nice. 😎 

Would you also accept a method like this?

```ruby
def runtime = mono_now - @_start_time
```

Very handy, e.g. in combination with my other suggestion.

Reply:
Is that runtime or wall time? Iterable jobs can be interrupted and sit on the queue, not executing.